examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -57,7 +57,7 @@
     "HugeCTR is a GPU-accelerated recommender framework designed to distribute training across multiple GPUs and nodes and estimate Click-Through Rates (CTRs).<br>\n",
```

Comment:
<p>We need to move them there for inference. Can we add the reason why we move them to it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>rephrased this explaining the situation to the reader</p>